### ISSUE TITLE
Job for crave_n1.service failed because the control process exited with error code

### ISSUE THREAD

**Issue by tregismoreira** [masternodes/vps#53]
I've set up my VPS and installed the first masternode as CRAVE. But once I execute the script to start the masternode, I'm getting the following error:

```
Created symlink from /etc/systemd/system/multi-user.target.wants/crave_n1.service to /etc/systemd/system/crave_n1.service.
Job for crave_n1.service failed because the control process exited with error code. See "systemctl status crave_n1.service" and "journalctl -xe" for details.
```

This issue seems related to #5. I've already set up the server from scratch but still got the same error.

Can you help me?

**Comment by tregismoreira** [comment]
Here are the results of the suggested commands:

`systemctl status crave_n1.service`
```
● crave_n1.service - crave distributed currency daemon
   Loaded: loaded (/etc/systemd/system/crave_n1.service; enabled; vendor preset: enabled)
   Active: activating (auto-restart) (Result: exit-code) since Thu 2018-01-25 00:28:33 UTC; 699ms ago
  Process: 9184 ExecStart=/usr/local/bin/craved -daemon -pid=/var/lib/masternodes/crave1/crave.pid -conf=/etc/masternodes/crave_n1.conf -datadir=/var/lib/masternodes/crave1 (code=ex
lines 1-4/4 (END)
```

`journalctl -xe`
```
-- Defined-By: systemd
-- Support: http://lists.freedesktop.org/mailman/listinfo/systemd-devel
--
-- Unit crave_n1.service has begun starting up.
Jan 25 00:29:36 masternodes01 systemd[9246]: crave_n1.service: Failed at step EXEC spawning /usr/local/bin/craved: No such file or directory
-- Subject: Process /usr/local/bin/craved could not be executed
-- Defined-By: systemd
-- Support: http://lists.freedesktop.org/mailman/listinfo/systemd-devel
--
-- The process /usr/local/bin/craved could not be executed and failed.
--
-- The error number returned by this process is 2.
Jan 25 00:29:36 masternodes01 systemd[1]: crave_n1.service: Control process exited, code=exited status=203
Jan 25 00:29:36 masternodes01 systemd[1]: Failed to start crave distributed currency daemon.
-- Subject: Unit crave_n1.service has failed
-- Defined-By: systemd
-- Support: http://lists.freedesktop.org/mailman/listinfo/systemd-devel
--
-- Unit crave_n1.service has failed.
--
-- The result is failed.
Jan 25 00:29:36 masternodes01 systemd[1]: crave_n1.service: Unit entered failed state.
Jan 25 00:29:36 masternodes01 systemd[1]: crave_n1.service: Failed with result 'exit-code'.
Jan 25 00:29:41 masternodes01 systemd[1]: crave_n1.service: Service hold-off time over, scheduling restart.
Jan 25 00:29:41 masternodes01 systemd[1]: Stopped crave distributed currency daemon.
-- Subject: Unit crave_n1.service has finished shutting down
-- Defined-By: systemd
-- Support: http://lists.freedesktop.org/mailman/listinfo/systemd-devel
--
-- Unit crave_n1.service has finished shutting down.
Jan 25 00:29:41 masternodes01 systemd[1]: crave_n1.service: Start request repeated too quickly.
Jan 25 00:29:41 masternodes01 systemd[1]: Failed to start crave distributed currency daemon.
-- Subject: Unit crave_n1.service has failed
-- Defined-By: systemd
-- Support: http://lists.freedesktop.org/mailman/listinfo/systemd-devel
--
-- Unit crave_n1.service has failed.
--
-- The result is failed.
Jan 25 00:29:48 masternodes01 kernel: [UFW BLOCK] IN=ens3 OUT= MAC=56:00:01:59:2c:9c:fe:00:01:59:2c:9c:08:00 SRC=222.161.223.54 DST=45.76.244.36 LEN=40 TOS=0x00 PREC=0x00 TTL=52 ID=
Jan 25 00:30:05 masternodes01 kernel: [UFW BLOCK] IN=ens3 OUT= MAC=56:00:01:59:2c:9c:fe:00:01:59:2c:9c:08:00 SRC=5.188.86.51 DST=45.76.244.36 LEN=40 TOS=0x08 PREC=0x20 TTL=240 ID=42
```

**Comment by marsmensch** [comment]
This is a regression inherited from the codebase switch last week. Fixed with 30acb3a69e7a794df8b7a93efca46448ac1ae565

Update the nodemaster script with a git pull and you will receive the changes. Sorry for the issue!

**Comment by tregismoreira** [comment]
Great, @marsmensch. It's working sweetly now. Thanks! :)
